src/software/thunderscope/gl/widgets/icon_loader.py

```diff
@@ -0,0 +1,18 @@
+from pyqtgraph.Qt import QtGui
+import os
+
+
+def get_icon(path: os.PathLike, color: str = None) -> QtGui.QPixmap:
```

Comment:
Unrelated, but have we thought about using an icon library in tscope (e.g. https://github.com/spyder-ide/qtawesome)?